.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Upload coverage
         uses: ./
         with:
-          coverageEndpoint: ${{ secrets.COVERAGE_ENDPOINT }}
+          coverageEndpoint: http://a7b7-75-159-190-90.ngrok.io/api/coverage
```

Comment:
```suggestion
          coverageEndpoint: ${{ secrets.COVERAGE_ENDPOINT }}
```

Reply:
im testing Ill re request for review after chef